### ISSUE TITLE
Referencing GlobalConfiguration.Configuration in OWIN integration

### ISSUE THREAD

**Issue by dotnetjunkie** [autofac/Autofac.WebApi.Owin#5]
The [Web API integration docs](http://docs.autofac.org/en/latest/integration/webapi.html) state:

> A common error in OWIN integration is use of the GlobalConfiguration.Configuration. In OWIN you create the configuration from scratch. You should not reference GlobalConfiguration.Configuration anywhere when using the OWIN integration.

The documentation however doesn't explain it's a bad idea to reuse the `GlobalConfiguration.Configuration`. Can you elaborate on this, or reference a source that explains the problems with this?

**Comment by tillig** [comment]
One of the most common questions we see on StsckOverflow and other venues goes basically like this: "I had a Web API site wired up without OWIN using GlobalConfiguration. I followed the various examples in getting started blog posts and created my own HttpConfiguration and configured some things in that but left others still using GlobalConfiguration. I wired Autofac to GlobalConfiguration and never changed that. Basically nothing in my site works anymore. Why not?"

We also see "I switched totally to OWIN and create my own configuration but have controllers that are using GlobalConfiguration for service location and that's broken. What gives?"

While it's _technically possible_ if you're _very careful and mindful_ to use GlobalConfiguration in OWIN, the majority of the "this doesn't work" questions and issues come from that. Hence the guidance - let us help you fall into the pit of success. Stop doing things that will potentially catch you up with subtle gotchas.
